tests/foreman/api/test_docker.py

```diff
@@ -453,9 +452,31 @@ def test_add_multiple_docker_repos_to_content_view(self):
 
         @Feature: Docker
 
-        @Status: Manual
-
         """
+        upstream_name = u'busybox'
+        product = entities.Product(organization=self.org).create()
+        repos_amount = randint(1, 5)
+        repos = [
+            _create_repository(product, gen_string('alpha'), upstream_name)
+            for _
+            in range(repos_amount)
+        ]
+        self.assertEqual(len(repos), repos_amount)
```

Comment:
This assertion does nothing useful. It just asserts that a loop - part of the Python language - worked correctly.